templates/dumps.yaml

```diff
@@ -1,6 +1,3 @@
-# Only run this if the ConfigMaps don't already exist
-{{- if and (not (lookup "v1" "ConfigMap" .Release.Namespace "configdb-json-dumps")) (not (lookup "v1" "ConfigMap" .Release.Namespace "auth-json-dumps")) }}
-
```

Comment:
@amrc-benmorrow Any idea why this was removed? This whole thing is a candidate for `acs-service-setup`.

Reply:
https://github.com/AMRC-FactoryPlus/amrc-connectivity-stack/commit/5183992396a9b0af112147b5396337feafe2b0c5

Reply:
I was having problems with them not getting updated when running `helm upgrade`, because they already existed and so Helm was skipping the whole file.

Reply:
I think the account UUID generation should be able to be handled by krbkeys now. If we create appropriate (fixed UUID) Service Requirement groups, to which permissions are granted in the dump file (or by service-setup), then krbkeys can now create fresh (per-deployment UUID) accounts and place them in the groups.